.gitignore

```
@@ -24,3 +24,4 @@ htmlcov/
 .pytest_cache/
 cupy/.data/
 cupy/cuda/thrust.h
+.vscode/
```

Could you remove unrelated change?

```suggestion
.vscode/
```